.github/workflows/update_codeowners.yml

```diff
@@ -0,0 +1,39 @@
+name: update codeowners
+on:
+  schedule:
+    - cron: 0 * * * *
```

Comment:
This is every hour, right? Seems a bit excessive, daily is enough?

Reply:
I worry about the situation where someone has a stack of PRs; they merge one and then the following one doesn't get flagged as needing additional review even though it touches interop files. 

I guess the problem is that because we aren't auto-merging the PR here the main latency is actually someone noticing and merging the update pull request. So maybe daily is fine, but probably the real solution is to auto-merge the updates.